chapters/offenses.adoc

```diff
@@ -1,152 +1,169 @@
 
-NOTE: It is understood that the ball may be bumped by the robot multiple times over a short distance while the kick is being taken. This is why a distance of 0.05 meters is used to decide whether a robot violates this rule or not. Remaining in contact with the ball for more than 0.05 meters also counts as double touch, even though technically the robot only touched the ball once.
+NOTE: If the referee is not sure which sanction to choose, he may confer with members of the <<Technical Committee, technical committee>> or the <<Organizing Committee, organizing committee>>.
 
-==== Attacker In Defense Area
-A robot must not touch the ball while being partially or fully inside the opponent <<Defense Area, defense area>>.
+Some examples of unsporting behavior are listed below.
 
-NOTE: When the ball is <<Ball In And Out Of Play, out of play>>, the more strict rule <<Robot Too Close To Opponent Defense Area>> applies instead.
+==== Damaging Other Robots
+It is not allowed to damage or modify robots of other teams.
 
-==== Excessive Dribbling
-A robot must not <<Dribbling Device, dribble>> the ball further than 1 meter, measured linearly from the ball location where the dribbling started. A robot begins dribbling when it makes contact with the ball and stops dribbling when there is an observable separation between the ball and the robot.
+==== Damaging The Field Or The Ball
+It is not allowed to damage or modify the field or the ball.
 
-NOTE: Dribblers can still be used to dribble large distances with the ball as long as the robot periodically loses possession, such as kicking the ball ahead of it as human soccer players often do.
+==== Showing Lack Of Respect
+A team member must show appropriate respect to everyone involved in the game. Infringements of this rule include but are not limited to:
 
-==== Ball Speed
-A robot must not accelerate the ball faster than 6.5 meters per second in 3D space.
+* insulting the opponent, the <<Referee, referee>> or other persons holding an <<Impartial Roles, impartial role>>
+* annoying the <<Referee, referee>> or other persons holding an <<Impartial Roles, impartial role>>
+* not obeying the orders of the <<Referee, referee>>
 
 === Fouls
-Fouls are violations of the rules that result in a <<Free Kick, free kick>> for the opposing team. The free kick will be shot from the position where the ball was located when the offense began happening (see <<Free Kick, the free kick rules>> for the exact ball position rules). If the foul happened while the ball is <<Ball In And Out Of Play, out of play>>, no free kick is given.
+The number of fouls per team is tracked by a counter. Each foul will
+increase the counter by one. Every third increase to the foul counter
+causes a <<Yellow Card, yellow card>> to be awarded.
 
-Every third foul of the same team results in a <<Yellow Card, yellow card>>.
+Every violation in this section and its subsections increase the foul
```

Comment:
This is not technically true since multiple defender does not increase the foul counter. It is however very explicitly written there that it does not

Reply:
I've added a suggestion to formulate it more correctly.